picard/ui/coverartbox.py

```diff
@@ -550,6 +542,12 @@ def contextMenuEvent(self, event):
             use_orig_value_action.triggered.connect(self.item.keep_original_images)
             menu.addAction(use_orig_value_action)
 
+        if self.item and self.item.can_show_coverart:
+            name = _('Choose local file ...')
+            choose_local_file_action = QtWidgets.QAction(name, self.parent)
+            choose_local_file_action.triggered.connect(self.choose_local_file)
+            menu.addAction(choose_local_file_action)
+
```

Comment:
The options (defined below) to Replace or Append "on drop" should be renamed since it seems they also apply to the file set through this dialog (and that's not a "drop").

Reply:
I just removed the "on drop" part, so now it just reads "Replace front cover art" and "Append front cover art". I think this is clear enough for a menu action, everything else requires a longer explanation text.